docs/topics/modules/api_get_component_analyses_response.adoc

```diff
@@ -26,7 +24,7 @@ None
 == Example request with cURL
 
 ----
-curl -k -H "Authorization: Bearer <openshift.io authentication token>" https://<recommender api url>/api/v1/component-analyses/maven/io.vertx:vertx-core/3.4.1
+curl https://<3scale APICast Gateway URL>/component-analyses/maven/io.vertx:vertx-core/3.4.1?user_key=<user-key>
```

Comment:
https://<3scale APICast Gateway URL>/component-analyses/maven/io.vertx:vertx-core/3.4.1?user_key=<user-key>
should be 
https://<3scale APICast Gateway URL>/api/v1/component-analyses/maven/io.vertx:vertx-core/3.4.1?user_key=<user-key>

Reply:
@samuzzal-choudhury can you please confirm on this?

Reply:
@Preeticp I confirm. Let's go with @invincibleJai 's suggestion.

Reply:
@Preeticp  `https://<3scale APICast Gateway URL>/api/v1/component-analyses/maven/io.vertx:vertx-core/3.4.1?user_key=<user-key>`